docs/BuildConfiguration.md

```diff
@@ -4,44 +4,45 @@ This table lists all the JSON properties that can be defined in a `build-config.
+| `staticBikeParkAndRide`            | Whether we should create bike P+R stations from OSM data                                                                                                                                                                                                                         | boolean        | false                   |                                                                                           |
+| `staticParkAndRide`                | Whether we should create car P+R stations from OSM data                                                                                                                                                                                                                          | boolean        | true                    |                                                                                           |
+| `streets`                          | Include street input files (OSM/PBF)                                                                                                                                                                                                                                             | boolean        | true                    |                                                                                           |
+| `storage`                          | Configure access to data sources like GRAPH/OSM/DEM/GTFS/NETEX/ISSUE-REPORT.                                                                                                                                                                                                     | object         | null                    |                                                                                           |
+| `subwayAccessTime`                 | Minutes necessary to reach stops served by trips on routes of `route_type=1` (subway) from the street                                                                                                                                                                            | double         | 2.0                     | units: minutes                                                                            |
+| `timeZone`                         | Time zone for the graph. This is used to store the timetables in the transit model, and to interpret times in incoming requests. This is required if the inout data contains agencies with multiple timezones. If empty it will use the time zone shared by all of the agencies. | string         | null                    | The value can be given either as a zone id, or an UTC offset.                             |
```

Comment:
```suggestion
| `timeZone`                         | Time zone for the graph. This is used to store the timetables in the transit model, and to interpret times in incoming requests. This is required if the input data contains agencies with multiple timezones. If empty it will use the time zone shared by all of the agencies. | string         | null                    | The value can be given either as a zone id, or an UTC offset.                             |
```